student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom, age, name = 'Unknown', parents_permission: true)
+    super(age, name, parents_permission: parents_permission)
```

Comment:
- [ ] As you have done with the name it is not necessary to declare a default value to parent permission again, please 🙏🏽 remove the default value for the correct syntax.